collections/_article/godot-foundation-update-sep-2024.md

```diff
@@ -0,0 +1,98 @@
+maintain very high standards.
+
+Recently we adopted a modified version of those policies that is more specific to our organization. You can read the
+updated version on the [Godot Foundation website](https://godot.foundation/policies-and-procedures/key-policies).
+
+We are very happy with the policies and we hope you are too. They hold us to a very high standard and ensure that the
+Foundation will always operate in a way that supports Godot. 
+
+## Contractors
+The Foundation has 9 contractors working on the engine directly (5 full time, 4 part time) and 3 contractors
+working on administration + infrastructure right. We are currently in the hiring process for 1-2 more positions.
+
+Even with 12 people we feel very understaffed as there is always more work that can be done. Accordingly, we would like
+to continue hiring once funding allows for it.
+
+## Upcoming projects
+The Foundation is working on 4 major projects in addition to working on Godot itself:
+1. An official asset store
+2. An official job board on the Godot website
+3. A public project priorities document
+4. A public trademark policy
+
+### Asset Store
+We have been working towards an initial version of the asset store for a while now. It has taken us some time to get the
+legal aspects into place and a design agreed upon. 
+
+We will have a dedicated announcement soon with more information!
+
+### Job Board
+We aim to have a place for people to post that are looking to hire Godot developers to add to their game team or Godot
+contributors to fix high priority bugs that may be impacting your team.
+
+In addition to a job board, we will be curating a list of service providers who have shown that they offer services in
+the Godot Ecosystem to make it easy to find someone to contract with if you need support or any other services.
+
+Much of the foundational work for this has been finished, but it will likely be released after the asset store so we can
+focus our efforts more on the asset store. 
+
+### Project Priorities
+Soon we will be adding a project priorities page to our website to give the community more clarity on the direction of
+development.
+
+We have avoided providing a roadmap as we don’t want to promise more than we can deliver, or put too much pressure on
+volunteer contributors. However, the Godot Project maintainers have agreed that it is time to at least write down what
+the largest tasks are that we are working towards. This won’t be like a traditional roadmap where we have specific tasks
+with expected dates. However, we believe it will answer a lot of questions that the community has about the future of
+the project.
+ 
+### Trademark Policy
+We registered the Godot trademark and the Godot logo trademark while still a member project of the SFC. Last year the
+SFC granted the Godot Foundation ownership over the trademarks. Now we need to have a public policy that says how and
+when others can use the trademarks in order to protect them.
```

Comment:
"In order" is generally redundant, so it can be removed:

```suggestion
We registered the Godot trademark and the Godot logo trademark while still a member project of the SFC. Last year, the
SFC granted the Godot Foundation ownership over the trademarks. Now we need to have a public policy that says how and
when others can use the trademarks to protect them.
```